frontend/src/metabase/admin/settings/auth/components/AuthCard/AuthCard.tsx

```diff
@@ -19,8 +19,8 @@ import {
   CardTitle,
 } from "./AuthCard.styled";
 
-export type AuthSetting = Omit<SettingDefinition, "value"> & {
-  value: boolean | null;
+export type AuthSetting = Omit<SettingDefinition, "value" | "key"> & {
```

Comment:
it never uses `key`

Reply:
I'd rather revert this change and add a key to `frontend/src/metabase/admin/settings/auth/components/AuthCard/AuthCard.unit.spec.tsx`

e.g.
![image](https://github.com/user-attachments/assets/ec7712b3-17de-427e-a078-fdbd6201d237)

Since we're actually passing a setting with an actual `key` here. This fix is just to fix the problem we have in unit testing.